administrator/language/en-GB/com_users.ini

```
@@ -188,7 +188,7 @@ COM_USERS_MAIL_PLEASE_FILL_IN_THE_FORM_CORRECTLY="Please fill in the form correc
 COM_USERS_MAIL_PLEASE_FILL_IN_THE_MESSAGE="Please enter a message"
 COM_USERS_MAIL_PLEASE_FILL_IN_THE_SUBJECT="Please enter a subject"
 COM_USERS_MAIL_PLEASE_SELECT_A_GROUP="Please select a Group"
-COM_USERS_MAIL_REMINDER_DESC="This mail is sent to a user when by the \"Forgot your username?\" link e.g. in a login form."
+COM_USERS_MAIL_REMINDER_DESC="This mail is sent to a user by the \"Forgot your username?\" link e.g. in a login form."
```

```suggestion
COM_USERS_MAIL_REMINDER_DESC="This mail is sent to a user by using the \"Forgot your username?\" link e.g. in a login form."
```
To make it consistent with COM_USERS_MAIL_PASSWORD_RESET_DESC.